### PR TITLE
Integrate updater settings into `SettingsView`

### PR DESCRIPTION
**Pull request by marcusziade**
- This implements the new planned updater settings user interface by integrating it in the `SettingsView`
- macOS native `cmd+,` now opens the `SettingsView`
- `UpdaterSettingsView.swift` has been removed from the project as it's now redundant code

image
image

### CODE REVIEW

**Comment by jeremybosma** [comment]
This looks really good, although we are acc trying to move this into the native settings page on Mythic at the very bottom under epic in a tab named Updater Settings. We haven't started on this yet so let us know if you're down to change your pr to do that and we'll be glad to merge.

**Comment by jeremybosma** [comment]
It's mostly copy paste work i believe since you already did the visual and the frontend already exists. Perhaps even make the settings in menu bar and keybind redirect to the settings page. This way we will not have two settings views. What do you think?

**Comment by marcusziade** [comment]
Alright. i'll do that. could we do that in a separate PR. Should be fine to just iterate more, right?

**Comment by marcusziade** [comment]
Nevermind, I see what you mean. I'll do it here

**Comment by marcusziade** [comment]
Yeah, having the default cmd+, open this view could be good 
CleanShot 2024-06-21 at 07 24 16

**Comment by marcusziade** [comment]
@jeremybosma done

File: Mythic/Views/Navigation/Settings.swift [hunk 1]
```diff
@@ -221,12 +225,21 @@ struct SettingsView: View {
...
+                .padding(.horizontal)
+            }
+            
             /* FIXME: TODO: Temporarily disabled; awaiting view that directly edits Wine.defaultBottleSettings.
             Section("Default Bottle Settings", isExpanded: $isDefaultBottleSectionExpanded) {
                 // BottleSettingsView something
             }
              */
         }
+        .frame(minHeight: 750)
```
Comment (marcusziade) [inline Mythic/Views/Navigation/Settings.swift:242]: this ensures no scrolling is needed on most displays

**Comment by marcusziade** [comment]
This issue is there on `main` as well. Not sure yet what causes that, but I think it's out-of-scope from this PR

CleanShot 2024-06-21 at 10 50 42

**Comment by jeremybosma** [comment]
Awesome, we’ll have to create an issue for that little bug. It looks great, awesome work!

**Comment by blackxfiied** [comment]
Perfect, but for the sake of conciseness, can we turn the SPUUpdater into an Environment variable or something similar? Jus t so we avoid scenarios like this:
```
SettingsView(
	  automaticallyChecksForUpdates: $automaticallyChecksForUpdates,
	  automaticallyDownloadsUpdates: $automaticallyDownloadsUpdates
)
```

**Comment by marcusziade** [comment]
Yes, you're right. Let's do that

**Comment by marcusziade** [comment]
@blackxfiied ✅  done https://github.com/MythicApp/Mythic/pull/110/commits/28c40b95ee9147c104ea602c6188778037102d8a

**Comment by marcusziade** [comment]
The CI will still fail here as this uses latest `main`

**Comment by blackxfiied** [comment]
Sadly, the CI is useless and this build is actually failing lol
Seems you forgot to include SparkleController.swift.

image

**Pull request merged**